### PR TITLE
Add wikibase:1.31

### PR DESCRIPTION
**Pull request by Ladsgroup**
Bug: T197212

### CODE REVIEW

**Comment by Ladsgroup** [comment]
The reason it fails is that the bundle can't find the base 1.31 image because it's not yet pushed to docker hub. I can split this PR to make sure first the base gets merged and then the bundle one.

**Comment by addshore** [comment]
Hmm,, but the first image should be tagged with wikibase/wikibase:1.31 when built? No?

File: wikibase/1.31/.travis/build-deploy.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#Oneline for full directory name see: https://stackoverflow.com/questions/59895/getting-the-source-directory-of-a-bash-script-from-within
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+set -e
+docker build --pull "$DIR/../base" -t wikibase/wikibase:1.31 -t wikibase/wikibase:1.31-base -t wikibase/wikibase:latest -t wikibase/wikibase:latest-base
```
Comment (addshore) [inline wikibase/1.31/.travis/build-deploy.sh]: the -t wikibase/wikibase:1.31 on this line should do that

**Comment by tarrow** [comment]
I think as @addshore suggested we should probably remove all the `--pull`s

**Pull request merged**